Cooper/ORM/EntityORM.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+
+namespace Cooper.ORM
+{
+    public class EntityORM
+    {
+        public List<AttributeValue> attributeValue {get;set;}
```

Comment:
make it a dictionary and you won't have the class AttributeValue. Dictionary is a perfect structure for it